deploy/deploy.go

```diff
@@ -510,6 +510,14 @@ func walkRemote(ctx context.Context, bucket *blob.Bucket) (map[string]*blob.List
 		if err != nil {
 			return nil, err
 		}
+		// Check include/exclude matchers.
+		slashpath := filepath.ToSlash(obj.Key)
+		if include != nil && !include.Match(slashpath) {
+			jww.INFO.Printf("  remote dropping %q due to include\n", slashpath)
```

Comment:
Neither of these appear to do anything other than log that it's doing something...?

Reply:
I've added continue statements here as well, do you think i should remove the logs?